HW8/TanchakYaroslav/task2_3/Modules_task2.py

```diff
@@ -0,0 +1,83 @@
+# Task 2 - Password Validation
+# Version 1 - without module re
+# Цю версію програми я написав до того як дойшов до моменту про регулярні вирази
+# Я думав як її оптимізувати але не поки не зміг нічого кращого придумати
+#
+import re
+
+alphabet_lower = "abcdefghijklmnopqrstuvwxyz"
+alphabet_upper = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+digits = "1234567890"
+symbols = "@#$"
+
+def password_validation(input_password):
```

Comment:
good job
look at this version of code
import re

password = input("Enter your password:\n")

if re.match("^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[$#@_])(?=.{6,16}$).*", password):
    print("Nice pass! Psss, I won't tell anyone ;)")
else:
    print("Password is not valid. Check if you have:"
          """
    At least 1 letter between[a-z] and 1 letter between[A-Z].
    At least 1 number between[0-9].
    At least 1 character from [$#@].
    Minimum length 6 characters.
    Maximum length 16 characters.
          """
           )